src/app/CommandHandler.cpp

```diff
@@ -573,7 +573,7 @@ FabricIndex CommandHandler::GetAccessingFabricIndex() const
     }
     else
     {
-        fabric = mpExchangeCtx->GetSessionHandle()->AsSecureSession()->GetFabricIndex();
+        fabric = mpExchangeCtx->GetSessionHandle()->GetFabricIndex();
```

Comment:
Please remove the branching here, since it's not neeed anymore.  This whole function should be:
```
  return mpExchangeCtx->GetSessionHandle()->GetFabricIndex();
```